examples/10K_Evaluator/10K_PDF_Summary.ipynb

```diff
@@ -0,0 +1,745 @@
+   "source": [
+    "---\n",
+    "\n",
+    "## 🚀 Process Overview\n",
+    "\n",
+    "We simplify the process into a single, streamlined step:\n",
+    "\n",
+    "- **Summary Extraction**:\n",
+    "  - Utilize Uniflow for parsing and summarizing PDF-formatted annual reports.\n",
+    "  - **Example Reports**: Nike, Amazon, and Alphabet serve as our case studies for demonstration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f59c9288-c716-49ab-82e9-a62d3f63f117",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "## 📋 Prerequisites\n",
+    "\n",
+    "> **Important**: A GPU is necessary for running Reinforcement Learning from Human Feedback (RLHF), a crucial component of the fine-tuning process.\n",
+    "\n",
+    "### Setup Instructions\n",
+    "\n",
+    "Ensure your environment is ready by following these steps:\n",
+    "\n",
+    "#### Conda Environment\n",
+    "\n",
+    "- Create a conda environment specific to this project. Follow the provided setup instructions for guidance.\n",
+    "\n",
+    "#### Installation of Dependencies\n",
+    "\n",
+    "Install necessary libraries within your environment:\n",
+    "\n",
+    "```bash\n",
+    "pip3 install uniflow\n",
+    "pip3 install \"pykoi[huggingface, rag, rlhf]\"\n",
+    "```\n",
+    "\n",
+    "For GPU support, ensure the correct version of torch is installed:\n",
+    "\n",
+    "```bash\n",
+    "pip3 uninstall torch\n",
+    "pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu121\n",
+    "```\n",
+    "\n",
+    "#### OpenAI API Key Configuration\n",
+    "\n",
+    "Obtain an OpenAI API key and set it as the `OPENAI_API_KEY` environment variable in a `.env` file at the root directory of this project. Refer to the provided [instructions](https://github.com/CambioML/cambio-cookbook/tree/main#api-keys) for more detail."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "319c8734-a752-44f6-9e1d-f951af75a898",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "## 🌟 Getting Started\n",
+    "\n",
+    "With your environment prepared and dependencies in place, you're ready to dive into the notebook. Follow the guide to start extracting and summarizing information from annual reports. This tool is designed to streamline the analysis of complex financial documents, making your review process more efficient and insightful."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37da3d68-4f3f-4232-9557-a6da710ce52c",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "### Update System Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "2c11a636-ebd0-4a3a-8dde-c37a5b1a1751",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%reload_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import sys\n",
+    "\n",
+    "sys.path.append(\".\")\n",
+    "sys.path.append(\"..\")\n",
+    "sys.path.append(\"../..\")"
```

Comment:
let's delete the system paths cell

Reply:
in the next cell, add `!{sys.executable} -m pip install -q uniflow==0.0.23` (i.e. we install uniflow directly from pip)